config/include.go

```diff
@@ -278,6 +278,12 @@ func (cfg *TerragruntConfig) Merge(sourceConfig *TerragruntConfig, terragruntOpt
 				cfg.Terraform.Source = sourceConfig.Terraform.Source
 			}
 
+			if sourceConfig.Terraform.CopyTerraformLockFile != nil {
+				cfg.Terraform.CopyTerraformLockFile = sourceConfig.Terraform.CopyTerraformLockFile
+			}
+
+			mergeExtraArgs(terragruntOptions, sourceConfig.Terraform.ExtraArgs, &cfg.Terraform.ExtraArgs)
```

Comment:
I believe there is an extra call to `mergeExtraArgs` here

Reply:
you are right, it was indeed duplicated, thanks for catching this up.